app/javascript/packages/document-capture/components/location-collection-item.tsx

```diff
@@ -32,56 +32,64 @@ function LocationCollectionItem({
-        </div>
-        <div>{streetAddress}</div>
-        <div>{formattedCityStateZip}</div>
-        {(weekdayHours || saturdayHours || sundayHours) && (
-          <h4>{t('in_person_proofing.body.location.retail_hours_heading')}</h4>
-        )}
-        {weekdayHours && (
-          <div>
-            {`${t('in_person_proofing.body.location.retail_hours_weekday')} ${weekdayHours}`}
+        <div className="grid-row">
+          <div className="grid-col-fill">
+            {numericDistance && (
+              <h3 className="usa-collection__heading margin-bottom-1">
+                {t('in_person_proofing.body.location.distance', {
+                  count: parseFloat(numericDistance),
+                })}
+              </h3>
+            )}
+            {!distance && <h3 className="usa-collection__heading margin-bottom-1">{name}</h3>}
+            <div>{streetAddress}</div>
+            <div>{formattedCityStateZip}</div>
+            {(weekdayHours || saturdayHours || sundayHours) && (
+              <h4>{t('in_person_proofing.body.location.retail_hours_heading')}</h4>
+            )}
+            {weekdayHours && (
+              <div>
+                {`${t('in_person_proofing.body.location.retail_hours_weekday')} ${weekdayHours}`}
+              </div>
+            )}
+            {saturdayHours && (
+              <div>
+                {`${t('in_person_proofing.body.location.retail_hours_sat')} ${saturdayHours}`}
+              </div>
+            )}
+            {sundayHours && (
+              <div>
+                {`${t('in_person_proofing.body.location.retail_hours_sun')} ${sundayHours}`}
+              </div>
+            )}
+            {phone && (
+              <div>
+                <h4>{t('in_person_proofing.body.location.contact_info_heading')}</h4>
+                <div>{`${t('in_person_proofing.body.location.phone')} ${phone}`}</div>
+              </div>
+            )}
+            <Button
+              id={`location_button_mobile_${selectId}`}
+              className="tablet:display-none margin-top-2 width-full heey"
+              onClick={(event) => handleSelect(event, selectId)}
+              type="submit"
+            >
+              {t('in_person_proofing.body.location.location_button')}
+            </Button>
           </div>
-        )}
-        {saturdayHours && (
-          <div>{`${t('in_person_proofing.body.location.retail_hours_sat')} ${saturdayHours}`}</div>
-        )}
-        {sundayHours && (
-          <div>{`${t('in_person_proofing.body.location.retail_hours_sun')} ${sundayHours}`}</div>
-        )}
-        {phone && (
-          <div>
-            <h4>{t('in_person_proofing.body.location.contact_info_heading')}</h4>
-            <div>{`${t('in_person_proofing.body.location.phone')} ${phone}`}</div>
+          <div className="grid-auto">
+            <Button
+              id={`location_button_desktop_${selectId}`}
+              className="display-none tablet:display-inline-block order-three"
```

Comment:
Do we need to use the `order` class helper here? I'd worry about its [accessibility implications](https://developer.mozilla.org/en-US/docs/Web/CSS/order#accessibility_concerns).

Reply:
> Do we need to use the `order` class helper here? I'd worry about its [accessibility implications](https://developer.mozilla.org/en-US/docs/Web/CSS/order#accessibility_concerns).

```suggestion
              className="display-none tablet:display-inline-block"
```

Reply:
oh no, that was a mistake (something older i was trying i meant to delete)